ckan/templates-midnight-blue/snippets/package_item.html

```diff
@@ -1,49 +1,51 @@
 {#
-Displays a single of dataset.
-
-package        - A package to display.
-item_class     - The class name to use on the list item.
-hide_resources - If true hides the resources (default: false).
-
-Example:
-
-  {% snippet 'snippets/package_item.html', package=c.datasets[0] %}
-
+  Displays a single of dataset.
+  
+  package        - A package to display.
+  item_class     - The class name to use on the list item.
+  hide_resources - If true hides the resources (default: false).
+  
+  Example:
+  
+    {% snippet 'snippets/package_item.html', package=c.datasets[0] %}
+  
 #}
 {% set title = package.title or package.name %}
 {% set notes = h.markdown_extract(package.notes, extract_length=180) %}
 
 {% block package_item %}
-  <li class="{{ item_class or "dataset-item" }}">
+  <li class="{{ item_class or 'dataset-item' }}">
     {% block content %}
       <div class="dataset-content">
         {% block heading %}
-          <h2 class="dataset-heading">
+          <h3 class="dataset-heading">
+            {% block heading_title %}
+              <a href="{{ h.url_for('%s.read' % package.type, id=package.name) }}" aria-label="{{ _('Navigate to dataset: {title}').format(title=title|truncate(80)) }}">
+                {{title|truncate(80)}}
+              </a>
+            {% endblock %}
             {% block heading_private %}
               {% if package.private %}
-                <span class="dataset-private badge bg-secondary">
-                    <i class="fa fa-lock"></i>
-                    {{ _('Private') }}
+                <span class="dataset-private badge bg-dust-red pull-right">
+                  {{ _('Private') }}
                 </span>
               {% endif %}
             {% endblock %}
-            {% block heading_title %}
-    <a href="{{ h.url_for('%s.read' % package.type, id=package.name) }}" aria-label="{{ _('Navigate to dataset: {title}').format(title=title|truncate(80)) }}">
-      {{title|truncate(80)}}
-    </a>
-            {% endblock %}
             {% block heading_meta %}
               {% if package.get('state', '').startswith('draft') %}
                 <span class="badge bg-info">{{ _('Draft') }}</span>
               {% elif package.get('state', '').startswith('deleted') %}
                 <span class="badge bg-danger">{{ _('Deleted') }}</span>
               {% endif %}
             {% endblock %}
-          </h2>
+          </h3>
         {% endblock %}
+        {% if package.organization %}
+          <p class="dataset-organization">Owner: {{ package.organization['title'] }}</p>
```

Comment:
"Owner" is an opinionated word. Let's just remove it and show just the org title (which should be a link to the org page)